### ISSUE TITLE
Druid rake damage

### ISSUE THREAD

**Issue by mofuggajnz** [Unlimited-WoW/BugTracker#148]
Hello this is mostly likely a suggestion.

I suggest to buff up rake dot damage by around 25% since in pvp it does irrevelant damage, and we all know that druid feral is a bleed class. Atm as full donor my rake would not hit over 30m.

Also druid for this reason are forced to play hibrid instead of a single spec. That makes it harder and just nonsense compared to other classes that just have to press 2 buttons for burst 300m in 1 second. ( aka dk, paladins, mages, priest, locks.)

So basically my suggestion is simple, buff a bit feral druid, it won't broke the game in any ways. Since they lack on energy regen anyways.

Main spells that should be buffed is Rake and Rip, but rake alone should fix the problem about druid ferals.

Thanks for the time on reading this and hope that this problem can be solved out.

**Comment by GladiatorUW** [comment]
Rake damage was tested and the output was confirmed by 10+ OG UW players back when we've adjusted spells.
